packages/vue/src/components/atoms/SfBadge/SfBadge.vue

```diff
@@ -1,5 +1,10 @@
-<template>
-  <div class="sf-badge">
+<template functional>
+  <div
+    :class="[data.class, data.staticClass, 'sf-badge']"
```

Comment:
where this `class` and `staticClass` come from?

Reply:
I added this data.class and data.staticClass to make this functional component fully transparent. Maybe it's not needed here? I just thought it would be better for DX.